packages/lesswrong/lib/collections/revisions/resolvers.js

```diff
@@ -110,7 +106,12 @@ addFieldsDict(Revisions, {
     resolveAs: {
       type: 'String',
       resolver: ({html}) => {
-        const mainTextHtml = sanitizeHtml(html, { nonTextTags: nonMainTextTags })
+        const mainTextHtml = sanitizeHtml(
+          html, { 
+            allowedTags: _.without(sanitizeHtml.defaults.allowedTags, 'blockquote'),
```

Comment:
Should use the same list sa in `vulcan-lib/lib/server/utils.js` (moving it to somewhere exported) rather than `sanitizeHtml.defaults.allowedTags`, since they might differ.